Source/Dafny/DafnyAst.cs

```diff
@@ -10224,6 +10224,31 @@ public MapComprehension(IToken tok, bool finite, List<BoundVar> bvars, Expressio
       TermLeft = termLeft;
     }
 
+    /// <summary>
```

Comment:
Curious, what tool processes these xml tags?

Reply:
Could we have a comment why being a general map comprehension is significant?  (Maybe just a copy of the detail of the PR, if that's the only significance.)

Reply:
These xml tags are used by the IDE (at least in Visual Studio, and I'm guessing also in Rider, but I haven't thought about it) to display hover text as information about the method.

The general map comprehensions were needed in some example where it was impossible to create the desired map with the simpler (and far more common) map comprehension. I think this is the example:

    map x | x in S :: f(x) := g(x)